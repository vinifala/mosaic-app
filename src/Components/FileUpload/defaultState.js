export const defaultState = {
  errorMessage: '',
  img: '',
};
