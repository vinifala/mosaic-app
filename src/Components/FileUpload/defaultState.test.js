import { defaultState } from './defaultState';

describe("File upload component's default state", () => {
  it('should match snapshot', () => {
    expect(defaultState).toMatchSnapshot();
  });
});
