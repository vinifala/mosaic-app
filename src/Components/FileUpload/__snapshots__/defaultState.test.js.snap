// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File upload component's default state should match snapshot 1`] = `
Object {
  "errorMessage": "",
  "img": "",
}
`;
